faketouches.js

```diff
@@ -16,6 +16,7 @@
         this.touches = [];
         this.touch_type = FakeTouches.TOUCH_EVENTS;
         this.has_multitouch = true;
+        this.use_prefixed_pointer_event_names = true;
```

Comment:
Might be better just to figure this out. Something like `'pointerdown' in document.body`?